scuba/__main__.py

```diff
@@ -207,6 +217,12 @@ def __locate_scubainit(self):
         if not os.path.isfile(self.scubainit_path):
             raise ScubaError('scubainit not found at "{}"'.format(self.scubainit_path))
 
+    @staticmethod
+    def get_paths_and_config():
+        '''Helper function to find and load .scuba.yml
+        '''
+        top_path, top_rel = find_config()
+        return top_path, top_rel, load_config(os.path.join(top_path, SCUBA_YML))
```

Comment:
It seems like this `staticmethod` would be better as a normal function, located in the `config`  module. 

And to avoid the redundant `os.path.join`, `find_config` could be updated to return a third item, the path to `.scuba.yml` itself.

Reply:
By updating `find_config` that way, we wouldn't need *any* extra functions.  Sounds like a fine idea to me.

Reply:
Addressed in TimothySprague/scuba@f48e2fd5